snmp/datadog_checks/snmp/data/conf.yaml.example

```
@@ -124,78 +138,72 @@ instances:
     #
     # retries: 5
 
-    ## @param tags - list of key:value element - optional
-    ## List of tags to attach to every metric, event and service check emitted by this integration.
-    ##
-    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
-    #
-    # tags:
-    #   - <KEY_1>:<VALUE_1>
-    #   - <KEY_2>:<VALUE_2>
-
     ## @param loader - string - optional - default: python
     ## Check loader to use. Available loaders:
-    ## - core: (recommended) will use new corecheck SNMP integration
-    ## - python: will use legacy python SNMP integration
+    ## - core: (recommended) Uses new corecheck SNMP integration
+    ## - python: Uses legacy python SNMP integration
     #
-    # loader: core
+    loader: core
 
-    ## @param oid_batch_size - integer - optional - default: 60 (core)
+    ## @param oid_batch_size - integer - optional - default: 60
     ## The number of OIDs handled by each batch. Increasing this number improves performance but
     ## uses more resources.
     ## Only available for corecheck implementation (`loader: core`).
+    ##
+    ## In corecheck, the default value depends on the version of the agent.
+    ## Before version 7.30 the default value is 60 and from version 7.30 its value is 5.
     #
-    # oid_batch_size: 10
+    # oid_batch_size: 60
```

```suggestion
    # oid_batch_size: 10
```

Let's keep same display value as before